### ISSUE TITLE
Give instructions on packaging to a single file for adding to path

### ISSUE THREAD

**Issue by Minion3665** [SpyHoodle/PrideFetch#13]
> Perhaps in a future commit we could make a "package.sh" script/similar? That would alleviate the complexity while also allowing us to cd relative to the script directory

_Originally posted by @Minion3665 in https://github.com/SpyHoodle/pridefetch/pull/12#discussion_r880965592_

**Comment by Minion3665** [comment]
From @SpyHoodle in the same comment thread
> Some repositories do typically have a `install.sh` instead of just a `package.sh`. I suggest we work & talk on this in the issue.

**Comment by Minion3665** [comment]
I suggest we don't have an `install.sh` unless we also intend to take steps to add to PATH in the same script; perhaps something like `build.sh` would be standard while also appropriate?

**Comment by SpyHoodle** [comment]
I can't see anyone wanting to build the package without installing it. Even if there was this would be a small subset of people, and it makes more sense to explain the building process for anyone that wants to do this. Originally I thought that we should make it install to PATH, but it seems like this would have many differences on systems, especially if someone ran it without even looking at it, which can happen often.

I still think, especially when we have packages for many distributions, we could add a "installing from source" section.

**Comment by Minion3665** [comment]
> I can't see anyone wanting to build the package without installing it.

I agree, however if our script doesn't install it I still don't think it's right to call it install.sh, even if that's the process that most users will be undertaking when they run the script. If build.sh is more accurate it should probably be called that

**Comment by SpyHoodle** [comment]
Definitely agree with you, sorry this wasn't clear.
